pkg/core/pipeline/main.go

```diff
@@ -217,11 +217,19 @@ func (p *Pipeline) runFlowCallback(d *dag.DAG, id string, depsResults []dag.Flow
 	logrus.Infof("\n%s: %s\n", leaf.Category, id)
 	logrus.Infof("%s\n", strings.Repeat("-", len(id)))
 
+	depsSourceIDs := []string{}
 	deps := map[string]*Node{}
 	for _, r := range depsResults {
 		if r.ID == rootVertex {
 			continue
 		}
+		if r.ID != rootVertex {
```

Comment:
That's not needed, we `continue` in the loop just above the condition is true

Reply:
Thanks, good catch